.github/CODEOWNERS

```diff
@@ -5,3 +5,7 @@
 *.js @newrelic/developer-enablement
 
 yarn.lock @newrelic/developer-enablement
+
+# Whats new files (pings individual project marketing managers)
+
+/src/content/whats-new @ishanmkh @jkinmonth @alexiskjones @molson10 @daynaslord
```

Comment:
the examples in the [github doc](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) use a trailing slash, so i think we should add one to this path:
```
src/content/whats-new/
```

Reply:
Should be fixed now @roadlittledawn